cli/doc/printer.rs

```diff
@@ -427,11 +427,23 @@ fn format_function_signature(node: &doc::DocNode, indent: i64) -> String {
 }
 
 fn format_class_signature(node: &doc::DocNode, indent: i64) -> String {
+  let class_def = node.class_def.clone().unwrap();
+  let super_suffix = if class_def.super_class.is_some() {
+    format!(
+      " {} {}",
+      colors::magenta("extends".to_string()),
+      colors::bold(class_def.super_class.unwrap())
+    )
+  } else {
+    String::from("")
+  };
```

Comment:
`if let` would be good for this:

```suggestion
  let super_suffix = if let Some(super_class) = class_def.super_class {
    format!(
      " {} {}",
      colors::magenta("extends".to_string()),
      super_class
    )
  } else {
    String::from("")
  };
```